packages/ui-components/src/components/loader/readme.md

```diff
@@ -0,0 +1,18 @@
+# kv-loader
```

Comment:
```suggestion
# *<kv-loader>*
```

and before you committing the code, you should run `npm run build:packages` to be able to have this Readme fully updated